src/content/id/_index.yaml

```diff
@@ -0,0 +1,241 @@
+# ATTENTION - changes made to this file not in GitHub will be over written!
+project_path: /web/_project.yaml
+book_path: /web/_book.yaml
+landing_page:
+  custom_css_path: /web/styles/wf-root.css
+
+  rows:
+
+  - heading: Pelajari Cara Mengembangkan Aplikasi Web Generasi Berikutnya
+    classname: devsite-landing-row-67
+    items:
+    - description: >
+        <p>
+          Pengguna rata-rata mengunjungi lebih dari 100 situs web di perangkat ponsel mereka
+          setiap bulan, dan berharap untuk mendapatkan kecepatan dan kualitas yang lebih tinggi
+          dari sebelumnya. Sumber daya di sini dapat membantu proyek Anda 
+          yang baru maupun yang sudah ada dengan generasi berikutnya dari teknologi web 
+          agar bisa memberikan konten yang cepat dan aman ke layar bagaimanapun.
```

Comment:
@tyohan, @petele, Since the WebFundamentals are re-designed using DevSite infrastructure, I prefer to implement the new DevSite infrastructure, then re-submit the Indonesian translations into several small PR per topics. What do you think?